indexd/index/drivers/alchemy.py

```diff
@@ -808,6 +808,10 @@ def update(self, did, rev, changing_fields):
                     )
                     for m_key, m_value in changing_fields['metadata'].items()]
 
+            if 'hashes' in changing_fields:
```

Comment:
Should we make this work more like updates to metadata/urls_metadata and delete any hashes that aren't given in the request? Like if the record already has SHA-256 and MD5, and the client just passes a new MD5, should we delete the old SHA-256?

Reply:
I started with overwrite, then switch to update. Thinking about it more I think I should make the behavior consistent with the metadata/urls_metadata.
TLDR I'm changing this to delete and overwrite.